mavros/src/plugins/waypoint.cpp

```diff
@@ -620,6 +628,8 @@ class WaypointPlugin : public plugin::PluginBase {
 		}
 	}
 
+	// @brief wait until a waypoint pull is complete
```

Comment:
/**
 * @brief wait until a waypoint pull is complete.
 * Pull happens asynchronously, this function blocks until it is done.
 */